lib/alchemy/page_layout.rb

```diff
@@ -151,11 +151,13 @@ def available_on_site?(layout)
       #
       def read_definitions_file
         if File.exist?(layouts_file_path)
-          YAML.safe_load(
-            ERB.new(File.read(layouts_file_path)).result,
-            permitted_classes: YAML_PERMITTED_CLASSES,
-            aliases: true,
-          ) || []
+          Array.wrap(
```

Comment:
@tvdeyen before I push the second commit of this, I want to make sure this seems like the right way to you.  In [page_layout_spec:30](https://github.com/AlchemyCMS/alchemy_cms/blob/a397ed242594a4b9f3ff38da6ac8751e5e9b2d7e/spec/libraries/page_layout_spec.rb#L22) it is expected to work when the yaml file returns a hash, not an array, so I'm converting to an array here.

Reply:
I think the spec is set up wrong. page layouts as well as elements should always be an array of hashes, but it is fine to ensure this here, if for some reason someone has a wrong yml file.